### PR TITLE
[Feature] IAP Manager Homepage

### LINKED ISSUE #9008: IAP manager homepage
### Details
This is a new page "Hire an IT apprentice" for the IAP. Page does NOT need to be translated into indigenous languages.

Suggested URL: talent.canada.ca/**indigenous-it-apprentice/hire**

contact the team email: [edsc.patipa.jumelage.emplois-itapip.job.matching.esdc@hrsdc-rhdcc.gc.ca](mailto:edsc.patipa.jumelage.emplois-itapip.job.matching.esdc@hrsdc-rhdcc.gc.ca)

### Blocked Design
Still missing from design:
- [x] Manager package - accessibility review and theme
- [x] Alt text review with @MCommonda and @RM-1978 
- [x] @substrae cleanup of figma file and dark mode if required

### Screenshot
image

### Figma file
https://www.figma.com/file/GPnW22jecfvifZRtWwZFTl/Hire-an-apprentice_Manager-Page_ITAP-edits?type=design&node-id=487-1879&mode=design&t=EhBHGqrADVbLglX3-0

### **Webp images**
https://drive.google.com/drive/folders/10fUq8m3vB4dv6YoFeHFlgLser7uquhMo?usp=sharing

### Acceptance criteria

- [ ] New page on IAP site (at suggested URL)
- [ ] New page matches Figma design and content
- [ ] Images use alt-text from Meagan in the comment below
- [ ] Images scaled to web standards
- [ ] New nav item on the IAP site "Hire an IT apprentice"
- [ ] Alt text for images: https://docs.google.com/document/d/1SyWi4Y1OGRjW7R5BYbL_fglZcrjDkPOflSrhWqAhLUg/edit?usp=sharing
- [ ] Download buttons should download this file: [ITAPIP Manager Package Text_FINAL_with frame.docx](https://github.com/GCTC-NTGC/gc-digital-talent/files/13939504/ITAPIP.Manager.Package.Text_FINAL_with.frame.docx)
- [ ] Contact the team link: edsc.patipa.jumelage.emplois-itapip.job.matching.esdc@hrsdc-rhdcc.gc.ca
- [ ] Add Chromatic snapshot 
- [ ] Dark mode tested/implemented?

### PR DESCRIPTION
**Pull request by petertgiles**
🤖 Resolves #9008

## 👋 Introduction

This branch adds a new IAP manager homepage.
h/t Vachan, Yoni, and Eric for pitching in!

## 🧪 Testing

1. Rebuild the site
2. Run storybook
3. Check the A/C in the issue

## 📸 Screenshot

image

### CODE REVIEW

**Comment by vd1992** [comment]
Did a bit of styling of the commitment & graduate section in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9042/commits/35cc4eac5ed98f90190812814ec90cdee81cc641
Two things I wasn't sure about: 

- I don't think the well colour shown in Figma is present in the Hydrogen config
- Is the image supposed to be cropped because the one in Figma appears to missing the bottom ~1/5

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,860 @@
...
+
+const placeholderUrl = "about:blank";
+const makeLink = (chunks: React.ReactNode, url: string) => (
+  <Link href={url} color="primary">
+    {chunks}
+  </Link>
+);
+
+function buildMailtToUri(
+  emailAddres: string,
```
Comment (vd1992) [inline apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:32]: Should be 
buildMailToUri (remove the 't' after 'Mail')
emailAddress (missing 's')
right?

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx [hunk 2]
```diff
@@ -0,0 +1,871 @@
...
+          <div data-h2-flex-grid="base(stretch, x3, x1) p-tablet(stretch, x2, x1) l-tablet(stretch, x3, x1)">
+            <div data-h2-flex-item="base(1of1) p-tablet(4of7)">
+              <Heading
+                data-h2-font-size="base(h3, 1)"
+                data-h2-margin="base(0, 0, x2, 0) p-tablet(0, 0, x2, 0)"
+                data-h2-text-align="base(center) p-tablet(left)"
+                data-h2-color="base(primary.darker)"
+              >
+                {intl.formatMessage({
+                  defaultMessage: "How the program works",
```
[the remark this answers is not in the document]
Comment (vd1992) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:335]: And needs a bit less spacing for the mobile view

Current

image

Figma

image

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,871 @@
...
+              >
+                <Heading
+                  data-h2-font-size="base(h3, 1)"
+                  data-h2-margin="base(0, 0, x1, 0)"
+                  data-h2-text-align="base(center) p-tablet(left)"
+                  data-h2-color="base(primary.darker)"
+                >
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "Indigenous talent ready for IT apprenticeships",
```
Comment (vd1992) [inline apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:579]: Little less spacing above for mobile view

Current

image

Figma

image

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx [hunk 4]
```diff
@@ -0,0 +1,871 @@
...
+                  id: "29o8/W",
+                  description:
+                    "Title of the 'A commitment to diverse digital talent' section",
+                })}
+              </Heading>
+              <p data-h2-margin="base(x1, 0, x1, 0)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The Digital Ambition, released in 2022, provides direction on how to increase representation of under-represented groups by leveraging programs like the IT Apprenticeship Program for Indigenous peoples.",
+                  id: "eSzizR",
```
Comment (vd1992) [inline apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:802]: I just noticed the underlining in the Figma file, is there supposed to be a link to something here?

image

**Review by vd1992 — COMMENTED** [review]
Okay! Looking good! Aside from stuff we've talked about, here are a few other things I found for your attention

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx (hunk 4 above)
Comment (petertgiles) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:792]: @gobyrne Do you have the link for this?

Comment (gobyrne) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:792]: EN: https://www.canada.ca/en/government/system/digital-government/digital-ambition.html
FR: https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/ambition-numerique.html

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx [hunk 5]
```diff
@@ -0,0 +1,871 @@
...
+                description: "Page title for IAP manager homepage",
+              })}
+            </h1>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "Together we can address barriers to Reconciliation, diversity and inclusion",
+                id: "mtLAap",
+                description: "Hero subtitle for IAP manager homepage",
+              })}
```
[the remark this answers is not in the document]
Comment (petertgiles) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:141]: Good catch.  I've updated it in [layout tweaks](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9042/commits/7f454ca005f12125d3f73a2854ef223b2abfc42b).

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx [hunk 6]
```diff
@@ -0,0 +1,871 @@
...
+                id: "sDqpzq",
+                description: "Call to action to download the manager's package",
+              })}
+            </Link>
+          </div>
+        </div>
+      </div>
+
+      {/* about the program section */}
+      <div data-h2-layer="base(1, relative)">
```
[the remark this answers is not in the document]
Comment (petertgiles) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:185]: Good catch.  I've updated the layout in [layout tweaks](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9042/commits/7f454ca005f12125d3f73a2854ef223b2abfc42b).  I still need to work on image zooming.

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (petertgiles) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx]: Good catch.  I've updated it in [layout tweaks](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9042/commits/7f454ca005f12125d3f73a2854ef223b2abfc42b).

**Review by gobyrne — APPROVED** [review]
English and French display well.

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (petertgiles) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:185]: I think the "zooming" is just the shifting aspect ratio.    I think the layout ended up slightly wider in the Figma design to the image appears a little shorter.

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx [hunk 7]
```diff
@@ -0,0 +1,871 @@
...
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* Testimonial section */}
+      <div
```
[the remark this answers is not in the document]
Comment (petertgiles) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:483]: I positied the trees better in [move testimonial background position](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9042/commits/9a6c5502ca4812abb3e56497f490dda87d10688b).
image

File: apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (vd1992) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx:141]: The font size doesn't appear changed though

image

[the remark this answers is not in the document]
Comment (petertgiles) [reply apps/web/src/pages/Home/IAPManagerHomePage/IAPManagerHomePage.tsx]: Oh, you meant the subtitle font size.  :facepalm:  Sorry, I missed that.
image
[fix subtitle font size](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9042/commits/9ad72092cd51c30d99fce568d13de6a6433beb28)

**Review by vd1992 — APPROVED** [review]

**Comment by substrae** [comment]
@petertgiles @vd1992 

Awesome work guys! This is looking really solid so far. Just a few small notes for you.

image

We can reduce the space between these sections slightly.

image

Let's increase the space above and below the testimonial text a bit.

image

Can we nudge the image down a bit so that more of the sky is showing?

image

Can we try and make these distances more even? The quotes are really close to the edge of the screen depending on your viewport.

image

Can we align the bottom of the quote with the bottom of the quote text, rather than the name of the person giving the testimonial? This was my own oversight in Figma (things moved around so much last minute, my bad!)

image

Can we make sure these distances are always equal?

image

We can increase this distance to match our regular spacing between `h2` headings (`x3` I believe)

**Comment by substrae** [comment]
@petertgiles @vd1992 

A couple notes for smaller viewports.

image

Can we align the first section with the rest of the sections padding-wise?

image

Can we center this button?

image

We can probably reduce the testimonial text size on smaller screens.

**Comment by brindasasi** [comment]
Looks very cool . 
Tiny thing : any reason reconciliation starting in caps?  

Screenshot 2024-01-23 at 2 34 10 PM

**Comment by petertgiles** [comment]
> Just a few small notes for you.

Thanks for the review, Josh.  I think I covered everything you noted.  Maybe we should adjust the base Button and Link components to center text for all buttons?

**Comment by petertgiles** [comment]
> Tiny thing : any reason reconciliation starting in caps?
Yeah, I thought that was strange, too.  That is how it is capitalized in the Figma design, so that's what I did.

**Pull request merged**